torch/testing/_internal/common_methods_invocations.py

```diff
@@ -18888,6 +18907,20 @@ def __init__(
             }), 'TestCommon', 'test_python_reference_consistency', device_type='cpu'),
         ),
     ),
+    PythonRefInfo(
+        "_refs.nn.functional.margin_ranking_loss",
+        torch_opinfo_name="nn.functional.margin_ranking_loss",
+    ),
+    PythonRefInfo(
+        "_refs.nn.functional.hinge_embedding_loss",
+        torch_opinfo_name="nn.functional.hinge_embedding_loss",
+        decorators=(
+            DecorateInfo(toleranceOverride({
```

Comment:
Once https://github.com/pytorch/pytorch/pull/78026 lands this decorator should no longer be necessary (unfortunately you'll have to rebase)